flytekit/core/map_task.py

```diff
@@ -76,7 +76,11 @@ def __init__(
 
         collection_interface = transform_interface_to_list_interface(actual_task.python_interface, self._bound_inputs)
         self._run_task: PythonFunctionTask = actual_task
-        _, mod, f, _ = tracker.extract_task_module(actual_task.task_function)
+        if hasattr(actual_task, "_IMPLICIT_OP_NOTEBOOK_TYPE"):
```

Comment:
This seems to special for notebook- we should make it for instance tasks in general